editor/src/components/navigator/navigator-item/navigator-item-wrapper.tsx

```diff
@@ -97,21 +98,31 @@ const elementSupportsChildrenSelector = createCachedSelector(
 )((_, navigatorEntry) => navigatorEntryToKey(navigatorEntry))
 
 export const labelSelector = createCachedSelector(
-  (store: MetadataSubstate) => store.editor.jsxMetadata,
+  (store: MetadataSubstate & ProjectContentSubstate) => store.editor.jsxMetadata,
```

Comment:
Looks like these can all be changed to the type alias created elsewhere in this PR.
```suggestion
  (store: ProjectContentAndMetadataSubstate) => store.editor.jsxMetadata,
```